usr/share/webapp-manager/firefox/profile/handlers.json

```diff
@@ -0,0 +1,26 @@
+{
+  "defaultHandlersVersion": {},
+  "mimeTypes": {
+    "application/pdf": { "action": 3, "extensions": ["pdf"] },
+    "image/webp": { "action": 3, "extensions": ["webp"] },
+    "image/avif": { "action": 3, "extensions": ["avif"] }
+  },
+  "schemes": {
+    "mailto": {
+      "stubEntry": true,
+      "handlers": [
+        null,
+        {
+          "name": "Gmail",
+          "uriTemplate": "https://mail.google.com/mail/?extsrc=mailto&url=%s"
+        }
+      ]
+    },
```

Comment:
I personally don't use gmail, so if this suddenly started running gmail for me, I'd be upset.